src/format/stream/stream_mut.rs

```diff
@@ -40,6 +40,15 @@ impl<'a> StreamMut<'a> {
 			av_stream_set_r_frame_rate(self.as_mut_ptr(), value.into().into());
 		}
 	}
+
+    pub fn set_codec_parameters_from(&mut self, context: &::codec::Context) -> Result<(), Error> {
```

Comment:
That's something I wanted but couldn't achieve because I couldn't figure out how to express it in rust myself.
Thanks for your suggestion.